dom.bs

```diff
@@ -7514,7 +7572,23 @@ string <var>value</var>, run these steps:
  <li><p>If <var>attribute</var>'s <a for=Attr>element</a> is null, then set <var>attribute</var>'s
  <a for=Attr>value</a> to <var>value</var>.
 
- <li><p>Otherwise, <a lt="change an attribute">change</a> <var>attribute</var> to <var>value</var>.
+ <li>
+  <p>Otherwise:
+
+  <ol>
+   <li><p>Let <var>originalElement</var> be <var>attribute</var>'s <a for=Attr>element</a>.
+
+   <li><p>Let <var>verifiedValue</var> be the result of calling <a>verify attribute value</a>
+   given <var>value</var>, <var>attribute</var>, and <a>this</a>.
+
+   <li><p>If <var>attribute</var>'s <a for=Attr>element</a> is null, then set <var>attribute</var>'s
+   <a for=Attr>value</a> to <var>value</var>, and return.
```

Comment:
Should this set Attr/value to `verifiedValue` (instead of `value`)?

WPT trusted-types/set-attributes-mutations-in-callback.tentative.html seems to expect verifiedValue here, and I get a number test failures if I implement this as written.

Reply:
Cc @fred-wang can you remember what we discussed. I seem to recall that maybe the tests for this specific function still need some work?

verifiedValue would make sense on the surface but there's also maybe a reason why it doesn't.

Reply:
(Apologies for the delay)

I just looked into this and I think it should indeed be verifiedValue. I'll update the spec PR now. Just as an fyi that test specifically is out of date with the spec currently so you'll probably still gets some failures.